shop/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+# H2
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
+# Datasource
+spring.datasource.url=jdbc:h2:~/Documents/App/online-shop-RaulDan/shop
```

Comment:
hello darkness my old friend. The code should be able to be run on any machine that has it, without further tweaks. therefore the path needs to be relative to the project, not to a folder on your device 📁